.github/workflows/osx.yml

```diff
@@ -12,8 +12,10 @@ jobs:
   build_osx:
     runs-on: macOS-latest
     steps:
-    - uses: actions/checkout@v1
-    - name: Build with dotnet
-      run: "./build.sh --ci /p:DisableNet461Tests=true"
+    - uses: actions/checkout@v2
+
+    - name: Update SubModules
+      run: git submodule update --init --recursive
```

Comment:
could you also update https://github.com/Azure/azure-signalr/blob/dev/docs/build-source.md#clone-the-source-code to include `submodule` setup similar to:
> Use --recursive to clone all submodules.
> `git clone https://github.com/Azure/azure-signalr --recursive`
> If you have already cloned, run this to initialize all submodules.
> `git submodule update --init`

Reply:
oO sorry i did not think about that, done